### PR TITLE
add feature/border

### PR DESCRIPTION
**Pull request by jmrodri610**
bla bla bla

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1,7 +1,8 @@
 body {
     background-color: aquamarine;
+    border: solid 3px fuchsia;
```
Comment (diana-moreno) [inline style.css]: Is not fuchsia, should be pink!

**Comment by jmrodri610** [comment]
ok, changed

File: style.css [hunk 2]
```diff
@@ -1,7 +1,8 @@
 body {
```
Comment (jmrodri610) [inline style.css:1]: ok

File: style.css [hunk 3]
```diff
@@ -1,7 +1,8 @@
...
     background-color: aquamarine;
+    border: solid 3px pink;
```
Comment (diana-moreno) [inline style.css:3]: ok

**Pull request merged**